fieldservice_person_salary/reports/fsm_order_template.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template
+        id="report_fsm_order_document_person_salary"
+        inherit_id="fieldservice.report_fsm_order_document"
+    >
+
+        <xpath expr="//div[@id='description']" position="after">
+            <div id="planning">
+                <h3>Planning</h3>
+                <div class="row">
+                    <div class="col-4">
+                        <span>Start Date: <span t-field="doc.scheduled_date_start"/></span>
+                    </div>
+                    <div class="col-4">
+                        <span>End Date: <span t-field="doc.scheduled_date_end"/></span>
+                    </div>
+                    <div class="col-4">
+                        <span>Duration: <span t-field="doc.scheduled_duration"/></span>
```

Comment:
Duration formatting. I think this widget doesn't require an extra addon dependency:
```suggestion
                        <span>Duration: <span t-field="doc.scheduled_duration" t-options="{'widget': 'duration', 'digital': True, 'unit': 'hour', 'round': 'minute'}"/></span>
```